tests/Analyzers/ThirdPartyAnalyzerFormatterTests.cs

```diff
@@ -210,6 +219,9 @@ void M()
 
                 // Prefer using. IDISP017
                 ["dotnet_diagnostic.IDISP017.severity"] = "error",
+
+                // We expect new line as end_of_line in this test
+                ["end_of_line"] = EndOfLineFormatter.GetEndOfLineOption("\n"),
```

Comment:
❔ Why was this added? This test has `expectedCode` that relies on the environment newline, and the default `end_of_line` value would be the same. This test wouldn't have been failing before this change, provided the user on Windows checked out the code with the _required_ `core.autocrlf=true` configuration.